### DISCUSSION TITLE
v108 and more method

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#317] [Announcements]
Exploit got updated, check it: https://github.com/3kh0/ext-remover#for-v106-and-up

Make a new discussion if you run into any problems!

**Comment by Alex2Mayhem** [comment]
> Exploit got updated, check it: https://github.com/3kh0/ext-remover#for-v106-and-up
> 
> Make a new discussion if you run into any problems!

We cannot do inspect element. :(

**Reply by 3kh0** [reply]
Run it as a bookmarklet

**Reply by Alex2Mayhem** [reply]
> Run it as a bookmarklet

107 Patched bookmarklets. :/

**Reply by 3kh0** [reply]
Check your chromeOS policies, javascript might have been blocked, otherwise your are fucked

**Reply by Alex2Mayhem** [reply]
> Check your chromeOS policies, javascript might have been blocked, otherwise your are fucked

They did. They did that in the 107 updates. as they still are.

**Reply by Alex2Mayhem** [reply]
also found a thing labeled key in the manifest.json

i did find a way how to know which teacher is connected with go guardian chrome-extension://haldlgldplgnggkjaafhelgiaglafanh/popup.html

**Reply by qDread-dev** [reply]
> Run it as a bookmarklet

IIRC bookmarklets are disabled on most if not all chrome pages (chrome-extension://, chrome://, ect)

**Reply by cainmik07** [reply]
Use the old link for ingot that take you to the google store error, works for me.

**Comment by ehunt34** [comment]
Stealing from TN again now are we?

**Reply by 3kh0** [reply]
TN tries to hide it, I think it should be free. No one has to join their shitty discord.

**Reply by cainmi** [reply]
v108.0.5359.172 doesn't work anymore, something is stopping cows from working and ingot never worked.

**Reply by Aka-but-nice** [reply]
Explain "cows" as it's used in your sentence.

**Reply by qDread-dev** [reply]
cows 💀

**Reply by obkduishinijf** [reply]
cows 💀 💀

**Reply by cainmik07** [reply]
> Explain "cows" as it's used in your sentence.

Compact Cow's UI

**Reply by cainmik07** [reply]
> cows 💀

Compact Cow's UI

**Reply by cainmik07** [reply]
> cows 💀 💀

Compact Cow's UI

**Reply by qDread-dev** [reply]
Its not called cows 💀

**Reply by cainmik07** [reply]
> Its not called cows 💀

It called Compact Cow's UI. Do y’all not know what an abbreviation is

**Reply by qDread-dev** [reply]
> > Its not called cows 💀
> 
> It called Compact Cow's UI. Do y’all not know what an abbreviation is

Its called ltbeef 💀 it doesnt matter what gui you use if at all

**Reply by cainmik07** [reply]
> > > Its not called cows 💀
> > 
> > 
> > It called Compact Cow's UI. Do y’all not know what an abbreviation is
> 
> Its called ltbeef 💀 it doesnt matter what gui you use if at all

And It’s still Compact Cow's UI, and ingot never work so I’m separating them, it does matter.

**Reply by qDread-dev** [reply]
Doesnt make sense that ingot wouldnt work but compact cows would, they are based off of the same thing

**Reply by cainmik07** [reply]
> Doesnt make sense that ingot wouldnt work but compact cows would, they are based off of the same thing

Dunno, it just didn’t work, it wouldn’t send the request to turn off but cow would, it doesn’t anymore because chrome updated but it used to be the only one that worked. It could be iboss or just the management that was on the chromebooks but it wouldn’t work.

**Reply by qDread-dev** [reply]
> > Doesnt make sense that ingot wouldnt work but compact cows would, they are based off of the same thing
> 
> Dunno, it just didn’t work, it wouldn’t send the request to turn off but cow would, it doesn’t anymore because chrome updated but it used to be the only one that worked. It could be iboss or just the management that was on the chromebooks but it wouldn’t work.

Interesting

**Comment by Aka-but-nice** [comment]
What is the point of this basically this solved nothing, if people had access to a inspect console and knowledge they would know how to pull this off, at least 79% of schools disable inspect and you can't run bookmarklets to mimic the inspect element on chrome-extension pages, so only about 21% of people can use this method. What's next you saying just to use dev mode, like ITS DISABLED,  ALL THE METHODS U LISTED ARE MOSTLY THINGS MOST PEOPLE CAN'T DO. The one the other day, it seemed to work, but who has access to a terminal, Who has money?
Most people here are all like 12-14

**Reply by Aka-but-nice** [reply]
Bro I have an idea so is there a inspect console command that interacts with policies or maybe accounts or settings, and you know how in the ext remover thing it uses the command "chrome.management.setEnabled" could you replace that command with a  command for like policies or settings that disables them

**Reply by qDread-dev** [reply]
> Bro I have an idea so is there a inspect console command that interacts with policies or maybe accounts or settings, and you know how in the ext remover thing it uses the command "chrome.management.setEnabled" could you replace that command with a command for like policies or settings that disables them

Interesting idea. However, there is not an equivalent for policies. At least not in the chrome.management category

**Reply by Aka-but-nice** [reply]
Dang, well how about either settings or accounts?

**Reply by md4r2** [reply]
Does anyone know of any inspect element bookmarklets with console built in? I used to have a few like xray goggles but they got discontinued...

**Reply by NightScripts-API** [reply]
> Does anyone know of any inspect element bookmarklets with console built in? I used to have a few like xray goggles but they got discontinued...

wont work

**Reply by 3kh0** [reply]
@md4r2 you could try firebug, there is a bookmark list on my website

**Reply by md4r2** [reply]
tried but for some reason js wont work on extension pages.
Also chrome://extensions is blocked by policy for me :|

**Reply by Aka-but-nice** [reply]
> tried but for some reason js wont work on extension pages. Also chrome://extensions is blocked by policy for me :|

Buddy, if chrome allowed bookmarklets on chrome://pages people "school" chromebooks, would basically be normal chromebooks

**Reply by Aka-but-nice** [reply]
> @md4r2 you could try firebug, there is a bookmark list on my website

Really trying to get view's eh'

**Reply by bartholemuew** [reply]
> > @md4r2 you could try firebug, there is a bookmark list on my website
> 
> Really trying to get view's eh'

my guy hes just being helpful

**Reply by fnfgo** [reply]
> What is the point of this basically this solved nothing, if people had access to a inspect console and knowledge they would know how to pull this off, at least 79% of schools disable inspect and you can't run bookmarklets to mimic the inspect element on chrome-extension pages, so only about 21% of people can use this method. What's next you saying just to use dev mode, like ITS DISABLED, ALL THE METHODS U LISTED ARE MOSTLY THINGS MOST PEOPLE CAN'T DO. The one the other day, it seemed to work, but who has access to a terminal, Who has money? Most people here are all like 12-14

dude 3kh0 is just trying to help man you couldnt figure this out by urself anyway so shut the fuck uo

**Reply by Aka-but-nice** [reply]
You stfu, you can't spell for the life of you, 
and I know at this point, it's not his fault.

**Comment by funkymokey** [comment]
I tried using point blank and doing the bookmark thing, but it was unable to be used? idk if i sound like a idiot or not but someone explain

**Reply by viperbot2000** [reply]
yea plz make one that works on school chromebooks

**Reply by Aka-but-nice** [reply]
> I tried using point blank and doing the bookmark thing, but it was unable to be used? idk if i sound like a idiot or not but someone explain

its because if you just copy the code it won't work you have to add javascript: then the code into a bookmark

**Comment by funkymokey** [comment]
Is there anything I can do with ssh? I'm not a coder but I have access to it

**Reply by Aka-but-nice** [reply]
wait, you have access, also for easier stuff, unlike ssh, try #347 or #348

**Reply by funkymokey** [reply]
None of those are lightspeed, if theres something i can do with ssh that works for lightspeed tell me please

**Reply by funkymokey** [reply]
ayo im still waiting

**Comment by DiscordThem3** [comment]
It wont let me inspect the page

**Reply by Aka-but-nice** [reply]
https://github.com/3kh0/ext-remover#106-and-higher-method

**Reply by DiscordThem3** [reply]
> https://github.com/3kh0/ext-remover#106-and-higher-method

ik but like i tried using firebug on the "chrome-extension://gndmhdcefbhlchkhipcnnbkcmicncehk/manifest.json" link but it wont let me inspect it, on other pages the inspect works but not on that 1

**Reply by Aka-but-nice** [reply]
bro look at the one that says 106 and higher method NOT ANY OTHER BRO

**Comment by DiscordThem3** [comment]
im not a coder so i wouldnt know but is there a way to get ingot on v106 up? and if so how long would it take to make

**Reply by qDread-dev** [reply]
No
Please stop asking

**Reply by Aka-but-nice** [reply]
Yes stop being retarded I just made one

**Reply by Aka-but-nice** [reply]
I'm making a pull request, you should see it soon...

**Reply by qDread-dev** [reply]
> Yes stop being retarded I just made one

Wow thats really cool!
Does it work in the same way as Ingot or do you mean a different unblock method?

**Reply by Alex2Mayhem** [reply]
> im not a coder so i wouldnt know but is there a way to get ingot on v106 up? and if so how long would it take to make

Look at the main page. scroll down and you'lll figure it out.

**Reply by Aka-but-nice** [reply]
> > Yes stop being retarded I just made one
> 
> Wow thats really cool! Does it work in the same way as Ingot or do you mean a different unblock method?

Look at <a href="https://github.com/3kh0/ext-remover">this</a> and click on "New Point Blank" if that doesn't work try Killcurly, which is also listed <a href="https://github.com/3kh0/ext-remover">here</a>

**Reply by qDread-dev** [reply]
> > > Yes stop being retarded I just made one
> > 
> > 
> > Wow thats really cool! Does it work in the same way as Ingot or do you mean a different unblock method?
> 
> Look at [this](https://github.com/3kh0/ext-remover) and click on "New Point Blank" if that doesn't work try Killcurly, which is also listed [here](https://github.com/3kh0/ext-remover)

Very interesting exploit, thanks!

**Reply by Aka-but-nice** [reply]
Np

**Reply by GlizxyBoi** [reply]
> Np

is there a bookmarklet for 106 and up?

**Reply by qDread-dev** [reply]
> > Np
> 
> is there a bookmarklet for 106 and up?
Are you able to read? He just said that there was

**Reply by cainmi** [reply]
glitch.me is blocked on mine can't do it.

**Reply by qDread-dev** [reply]
> glitch.me is blocked on mine can't do it.

crazy

**Reply by GlizxyBoi** [reply]
> Yes stop being retarded I just made one

wait then gimme the code then

**Reply by qDread-dev** [reply]
> > Yes stop being retarded I just made one
> 
> wait then gimme the code then

🤦🤦🤦🤦🤦🤦

**Comment by funkymokey** [comment]
Wheres the internet password one? i had forgotten the password and i need t

**Comment by GlizxyBoi** [comment]
can yall make a reading plus cheat for like the answers?

**Reply by qDread-dev** [reply]
You could do it

**Reply by GlizxyBoi** [reply]
> You could do it

i could do what? make the code for it?

**Reply by Aka-but-nice** [reply]
It's javascript, you find the source of whatever website you want to hack on, get the id's of elements or class then make a hack that modifies them

**Reply by qDread-dev** [reply]
> > You could do it
> 
> i could do what? make the code for it?

No, you could run for president
What do you think I'm saying that you could do? Of course that is what I am saying



> It's javascript, you find the source of whatever website you want to hack on, get the id's of elements or class then make a hack that modifies them

^^^^

**Reply by GlizxyBoi** [reply]
> It's javascript, you find the source of whatever website you want to hack on, get the id's of elements or class then make a hack that modifies them

idk how to

**Reply by GlizxyBoi** [reply]
> > > You could do it
> > 
> > 
> > i could do what? make the code for it?
> 
> No, you could run for president What do you think I'm saying that you could do? Of course that is what I am saying
> 
> > It's javascript, you find the source of whatever website you want to hack on, get the id's of elements or class then make a hack that modifies them
> 
> ^^^^

idk how

**Reply by qDread-dev** [reply]
> idk how

Crazy idea:
What if you figure it tf out?
You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!

**Reply by GlizxyBoi** [reply]
> > idk how
> 
> Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!

tf is google finna do💀

**Reply by GlizxyBoi** [reply]
> > idk how
> 
> Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!

dam sumones mad

**Reply by qDread-dev** [reply]
> > > idk how
> > 
> > 
> > Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!
> 
> dam sumones mad

All you do is beg people for help. You ask people for links to things that they posted 1 or 2 messages ago, you don't contribute anything, you not only have no knowledge, but also you do not even have the willingness to learn. Even though you have no knowledge of any programming language, with some simple googling you could have made what you wanted to already.

**Reply by GlizxyBoi** [reply]
> > > > idk how
> > > 
> > > 
> > > Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!
> > 
> > 
> > dam sumones mad
> 
> All you do is beg people for help. You ask people for links to things that they posted 1 or 2 messages ago, you don't contribute anything, you not only have no knowledge, but also you do not even have the willingness to learn. Even though you have no knowledge of any programming language, with some simple googling you could have made what you wanted to already.

i have knowledge ok but shit ill do sum shitty googling

**Reply by qDread-dev** [reply]
> 
> i have knowledge ok 
🧢🧢🧢

**Reply by funkymokey** [reply]
> > > > idk how
> > > 
> > > 
> > > Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!
> > 
> > 
> > dam sumones mad
> 
> All you do is beg people for help. You ask people for links to things that they posted 1 or 2 messages ago, you don't contribute anything, you not only have no knowledge, but also you do not even have the willingness to learn. Even though you have no knowledge of any programming language, with some simple googling you could have made what you wanted to already.

but why go through all this just let it go

**Reply by qDread-dev** [reply]
> > > > > idk how
> > > > 
> > > > 
> > > > Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!
> > > 
> > > 
> > > dam sumones mad
> > 
> > 
> > All you do is beg people for help. You ask people for links to things that they posted 1 or 2 messages ago, you don't contribute anything, you not only have no knowledge, but also you do not even have the willingness to learn. Even though you have no knowledge of any programming language, with some simple googling you could have made what you wanted to already.
> 
> but why go through all this just let it go

?

**Reply by obkduishinijf** [reply]
> > > > > idk how
> > > > 
> > > > 
> > > > Crazy idea: What if you figure it tf out? You constantly ask people for stuff, so I know you can type. That means you could use GOOGLE!!!!
> > > 
> > > 
> > > dam sumones mad
> > 
> > 
> > All you do is beg people for help. You ask people for links to things that they posted 1 or 2 messages ago, you don't contribute anything, you not only have no knowledge, but also you do not even have the willingness to learn. Even though you have no knowledge of any programming language, with some simple googling you could have made what you wanted to already.
> 
> i have knowledge ok but shit ill do sum shitty googling

"i have sum knowledge"
proceeds to show no proof

**Reply by qDread-dev** [reply]
>
> "i have sum knowledge" proceeds to show no proof

fr

**Reply by GlizxyBoi** [reply]
> > "i have sum knowledge" proceeds to show no proof
> 
> fr

y i need to show proof i have 1 C+ and rest r bs and As

**Reply by obkduishinijf** [reply]
> > > "i have sum knowledge" proceeds to show no proof
> > 
> > 
> > fr
> 
> y i need to show proof i have 1 C+ and rest r bs and As

imagine this, you get a 100% on a supposedly hard test, and your teacher wants you to retake it because she thinks you cheated, unless you can prove your work. how do you prove it? exactly, by doing the steps!

basically, proof is the only way someone can really know you did something.

**Reply by qDread-dev** [reply]
> > > "i have sum knowledge" proceeds to show no proof
> > 
> > 
> > fr
> 
> y i need to show proof i have 1 C+ and rest r bs and As

cap

**Reply by obkduishinijf** [reply]
> > > > "i have sum knowledge" proceeds to show no proof
> > > 
> > > 
> > > fr
> > 
> > 
> > y i need to show proof i have 1 C+ and rest r bs and As
> 
> cap

ong
